shared/api/storage_pool_volume_snapshot.go

```diff
@@ -30,6 +30,17 @@ type StorageVolumeSnapshotPost struct {
 	// New snapshot name
 	// Example: snap1
 	Name string `json:"name" yaml:"name"`
+
+	// Initiate volume snapshot migration
+	// Example: false
+	//
+	// API extension: storage_api_remote_volume_handling
```

Comment:
This isnt the API extension that added this field. Instead it should be the name of the new API extension you add in this PR.

Reply:
I was thinking a bit here but the concept of those extensions wasn't clear to me yet. Thanks. Just saw the client now also needs a check for `r.HasExtension`. Will add this in now.

Reply:
We use r.CheckExtension. Please could you add a comment to r.HasExtension that it is considered deprecated.

Reply:
Ok, done.